dist/src/main/java/io/camunda/application/commons/backup/DynamicIndicesConfiguration.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.application.commons.backup;
+
+import io.camunda.search.clients.DocumentBasedSearchClient;
+import io.camunda.search.connect.configuration.ConnectConfiguration;
+import io.camunda.webapps.backup.DynamicIndicesProvider;
+import io.camunda.webapps.profiles.ProfileWebApp;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ProfileWebApp
+public class DynamicIndicesConfiguration {
+
+  private static final String ELASTICSEARCH_TYPE = "elasticsearch";
```

Comment:
:wrench: there is an enum for it in the exporter module. I wonder if we shouldn't move it into the search module so it's commonly available, over plain string constants spread across the codebase.

Reply:
Probably yes, we currently have many instances of such enums scattered around the code base.

Reply:
I'll do it in a separate PR